pkg/solana/transmissions_cache.go

```diff
@@ -18,11 +16,7 @@ import (
 	"github.com/smartcontractkit/chainlink-solana/pkg/solana/logger"
 )
 
-var (
-	configVersion uint8 = 1
-)
-
-type ContractTracker struct {
+type TransmissionsCache struct {
```

Comment:
Probably out of scope, but would be interesting if the cache could be generic between the Transmissions and State cache?

Reply:
further extension would be generic caching utility moved to `chainlink-relay`?